### ISSUE TITLE
[Screen reader -Howdy-for-pms]: In scan mode narrator focus multiple times remains on Howdy home page link using down arrow key.

### ISSUE THREAD

**Issue by v-gasikk** [howdyai/howdy-website#21]
**User Experience**
People, with screen reader users, will face difficulty as the navigating to the control if narrator focus multiple time remains on same control.

**Test Environment**
OS: Win 10 Version 1809 (OS Build 17763.316)
URL: https://hcl-szz1925.howdy.ai/dashboard
Browser: Edge
Screen reader: Narrator 
 
**Repro Steps**
Turn on the narrator 
Step 1: Open the URL: https://hcl-szz1925.howdy.ai/dashboard
Step 2: Navigate to "Dashboard" tab and select it.
Step 3: Navigate to "Howdy for Your PM" link and select it.
Step 4: "Howdy for Your PM" page gets opened.
Step 5: Navigate to Howdy home page link by using down arrow key
Step 6: Observe the issue 

**Actual Result**
In scan mode narrator focus multiple times remains on Howdy home page link using down arrow key.

**Expected Result**
In scan mode narrator focus should moves to one time and should announce howdy home page link  info using down arrow key.

**Note**
Narrator is reading the unnecessary verbosity for search text box.
Please refer the attachment: Usability_Search text box.zip

[Usability_Howdy page.zip](https://github.com/howdyai/howdy-website/files/2973816/Usability_Howdy.page.zip)
[Usability_Search text box.zip](https://github.com/howdyai/howdy-website/files/2973817/Usability_Search.text.box.zip)

**Comment by benbrown** [comment]
This page has been deprecated and removed from the site.

**Comment by v-gasikk** [comment]
Now this page is not available. Closing this issue based on developer comment.
